browser/ui/views/brave_actions/brave_actions_container.cc

```diff
@@ -244,3 +246,7 @@ void BraveActionsContainer::OnExtensionActionUpdated(
     UpdateActionState(extension_action->extension_id());
 }
 // end ExtensionActionAPI::Observer
+
+void BraveActionsContainer::ChildPreferredSizeChanged(views::View* child) {
+  PreferredSizeChanged();
```

Comment:
Instead of calling `PreferredSizeChanged()`, just calling `parent()->Layout()` would be more explicit.
With this, we don't need to modify `BraveLocationBarView`.

Reply:
Ok, I got what you mean.